kedro/io/partitioned_dataset.py

```diff
@@ -379,7 +375,7 @@ class IncrementalDataset(PartitionedDataset):
         >>> dataset.load()
     """
 
-    DEFAULT_CHECKPOINT_TYPE = "kedro_datasets.text.TextDataSet"  # TODO: PartitionedDataset should move to kedro-datasets
+    DEFAULT_CHECKPOINT_TYPE = "kedro_datasets.text.TextDataset"  # TODO: PartitionedDataset should move to kedro-datasets
```

Comment:
```suggestion
    DEFAULT_CHECKPOINT_TYPE = "kedro_datasets.text.TextDataset"
```
Not sure what this comment is for, even though it's unrelated.

Reply:
Are we not moving PartitionedDataset anymore?

Reply:
Yes we are but we have tickets for that now, much easier to track than a random `TODO` 😅 :
https://github.com/kedro-org/kedro/issues/2713
and https://github.com/kedro-org/kedro-plugins/pull/253, which is ready for review too!